lib/compress.js

```diff
@@ -1053,9 +1053,7 @@ merge(Compressor.prototype, {
             var val = {};
             for (var i = this.properties.length; --i >= 0;) {
                 var prop = this.properties[i];
-                if (prop instanceof AST_ObjectKeyVal) {
-                    val[prop.key] = ev(prop.value, compressor);
-                }
+                val[prop.key] = ev(prop.value, compressor);
```

Comment:
@kzc no worries! I set aside this weekend to tackle this. I run into this case often enough to want to work on it. 😃